src/mlpack/methods/ann/loss_functions/CMakeLists.txt

```diff
@@ -0,0 +1,21 @@
+# Define the files we need to compile
+# Anything not in this list will not be compiled into mlpack.
+set(SOURCES
+  cross_entropy_error.hpp
+  cross_entropy_error_impl.hpp
+  KLDivergence.hpp
```

Comment:
This is a minor comment, but could we rename this `kl_divergence.hpp` during the merge (or now) to match the other filenames in mlpack?